languages/python/oso/tests/regression/test_unhandled_partial.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from typing import List
+
+def test_unhandled_partial():
```

Comment:
Super nit: update test name/comment/filename

Reply:
Updated a bit... this is a regression test so it doesn't really test anything too meaningful... Usually I'd just use issue numbers for this but we don't have that right now in notion. This better?